dangerfile.js

```diff
@@ -0,0 +1,120 @@
+const { markdown } = require('danger');
+const fs = require('fs').promises;
+const gzipSize = require('gzip-size');
+const git = require('simple-git/promise')(__dirname).silent(true);
+
+// https://stackoverflow.com/questions/15900485/correct-way-to-convert-size-in-bytes-to-kb-mb-gb-in-javascript
+const formatBytes = (bytes, decimals = 2) => {
+	if (bytes === 0) return '0 Bytes';
+
+	const k = 1000;
+	const dm = decimals < 0 ? 0 : decimals;
+	const sizes = ['B', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+
+	const i = Math.floor(Math.log(Math.abs(bytes)) / Math.log(k));
+
+	return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
+}
+
+const maybePlus = (from, to) => from < to ? "+" : "";
+
+const absDiff = (from, to) => {
+	if (from === to) {
+		return formatBytes(0);
+	}
+
+	return `${maybePlus(from, to)}${formatBytes(to - from)}`;
+}
+
+const percDiff =  (from, to) => {
+	if (from === to) {
+		return '0%';
+	}
+
+	return `${maybePlus(from, to)}${
+		(100 * (to - from) / (from || to)).toFixed(1)
```

Comment:
> from || to

How about we use a [proper formula](https://en.wikipedia.org/wiki/Relative_change_and_difference#Formulae)?

```suggestion
		(100 * (to - from) / Math.max(from, to)).toFixed(1)
```

Reply:
We're not looking for the larger of the two numbers here. We're using `||` because we want fallback should `from` be `0` but otherwise, we always want `from`.